site/public/js/rainbow-customization.js

```diff
@@ -267,78 +267,144 @@ function getGradeableBuckets() {
     return gradeables;
 }
 
-function getPlagiarism() {
-    const plagiarismData = [];
+/**
+ * Gets data to add to the JSON for the plagiarism and manual grade tables
+ * @param {int} table
+ *      0 -> plagiarism table
+ *      1 -> manual grading table
```

Comment:
Is there a reason why int values are useful here?  It might be more readable to instead pass strings which have at least a little bit of descriptive value.

Reply:
Refactored both functions to use string parameters. [966545c](https://github.com/Submitty/Submitty/pull/10653/commits/966545cb738aaa5febb0cb4923c1dc5ee6ba8bda)